scipy/spatial/tests/test_distance.py

```diff
@@ -1064,6 +1067,49 @@ def test_pdist_djaccard_random_nonC(self):
         Y_test2 = wpdist(X, 'test_jaccard')
         _assert_within_tol(Y_test2, Y_right, eps)
 
+    def test_pdist_jensenshannon_random(self):
+        eps = 1e-08
+        X = eo['pdist-double-inp']
+        Y_right = eo['pdist-jensenshannon']
+        Y_wrong = eo['pdist-jensenshannon-iris']
```

Comment:
Maybe I'm just missing something, but `Y_wrong` is used on this line and nowhere else in this test or elsewhere -- can you explain what it does?